inspire_dojson/hep/rules/bd2xx.py

```diff
@@ -129,9 +130,7 @@ def imprints2marc(self, key, value):
 
 @hep.over('preprint_date', '^269..')
 def preprint_date(self, key, value):
-    if isinstance(value, (tuple, list)):
-        return min(elem['c'] for elem in value if 'c' in elem)
-    return value.get('c')
+    return normalize_date(value.get('c'))
```

Comment:
This code was dealing with multiple `269__c` in the same `269` for no reason: there aren't any.